freshdesk_api.py

```diff
@@ -209,3 +214,30 @@ def api_endpoint(self, id):
 
     def create(self, id, **kwargs):
         return self.client.req(requests.post, self.api_endpoint(id), self.wrapper_name, **kwargs)
+
+class FreshDeskTickets(FreshDeskObjects):
+    api_name = 'ticket'
+    wrapper_name = 'helpdesk_ticket'
+
+    def convert_args_from_dic(self, args):
+        """
+        :param args :dic
+
+        :returns a string like company_id=14654687&filter_name=all_tickets
+        """
+        text_args = '?'
+        for key, value in args.items():
+            text_args = text_args+str(key)+'='+str(value)+'&'
+        return text_args[:-1]
+
+    def api_endpoint(self, id=None, args=None):
+        if id:
+            return '/helpdesk/{}s/{}.json'.format(self.api_name, id)
+        else:
+            if args:
+                # Add search params in the end of the URL
```

Comment:
Search params are already handled [here](https://github.com/oasiswork/python-freshdesk-client/blob/master/freshdesk_api.py#L42) (although the docstring don't explain what params do, I admit). Does it fit your need ?